src/containers/Header/NotificationsTray/index.jsx

```diff
@@ -0,0 +1,74 @@
+import styles from './style.postcss';
+
+import React, { PureComponent } from 'react';
+import Cursor from 'immutable-cursor';
+import Config from 'domain/Config';
+import moment from 'domain/moment';
+import connect from 'domain/connect';
+import Icon from 'components/Icon';
+import Callout from 'components/Callout';
+
+class NotificationsTray extends PureComponent {
+  constructor() {
+    super();
+    this.state = { open: false };
+    this._getRef = this._getRef.bind(this);
+  }
+
+  _getRef(node) {
+    this._node = node;
+  }
+
+  render() {
+    const { notifications } = this.props;
+    return <Callout content={
+      <div>
+        <div className={styles.NotificationsTray_popup_heading}>
+          <h2>Notifications</h2>
+        </div>
+        {(! notifications || ! notifications.size) ?
+          <div className={styles.NotificationsTray_popup_empty}>
+            <h1>Hello!</h1>
+            <p>You don't have any notifications yet.</p>
+            <aside>New notifications will appear here when teachers<br />
+              add internal notes for your students.</aside>
+          </div> : null}
```

Comment:
This could be written this way, too, which is slightly more beautiful:
```
{
  (! notifications || ! notifications.size) &&
  <div className={styles.NotificationsTray_popup_empty}>
    <h1>Hello!</h1>
    <p>You don't have any notifications yet.</p>
    <aside>New notifications will appear here when teachers<br />
      add internal notes for your students.</aside>
  </div>
}
```